### PR TITLE
Fixed smoothed screenshot

### LINKED ISSUE #31: Add API to take a screenshot

### PR DESCRIPTION
**Pull request by timocov**
**Type of PR:** bugfix

**PR checklist:**

- [x] Addresses an existing issue: fixes #31
- [ ] Includes tests
- [ ] Documentation update

### CODE REVIEW

File: src/gui/chart-widget.ts [hunk 1]
```diff
@@ -213,6 +213,7 @@ export class ChartWidget implements IDestroyable {
 		const targetCanvas = document.createElement('canvas');
 		resizeCanvas(targetCanvas, new Size(targetWidth, targetHeight));
 		const ctx = ensureNotNull(getContext2d(targetCanvas));
+		ctx.translate(0.5, 0.5);
```
Comment (timocov) [inline src/gui/chart-widget.ts]: It looks like this translation is already did in `resizeCanvas` function. Why do we need to have it twice?

Comment (timocov) [reply src/gui/chart-widget.ts]: @eugene-korobko GitHub treats me as PR's author and doesn't allow me request changes/approve/reject it. So, just come here please :)

**Comment by timocov** [comment]
Also, maybe we can enable hidpi in tests to handle this?

File: src/gui/chart-widget.ts (hunk 1 above)
Comment (eugene-korobko) [reply src/gui/chart-widget.ts]: Actually, we do not need translating at all, so we have to revert translate from `resizeCanvas`

File: src/gui/chart-widget.ts [hunk 2]
```diff
@@ -213,6 +213,8 @@ export class ChartWidget implements IDestroyable {
 		const targetCanvas = document.createElement('canvas');
 		resizeCanvas(targetCanvas, new Size(targetWidth, targetHeight));
 		const ctx = ensureNotNull(getContext2d(targetCanvas));
+		// we need this atually because of resizeCanvas does translate, but we do not need it
```
Comment (timocov) [inline src/gui/chart-widget.ts]:
```suggestion
		// we need this actually because of resizeCanvas does translate, but we do not need it
```

**Pull request merged**